packages/devtools_app/lib/src/shared/primitives/simple_items.dart

```diff
@@ -57,3 +57,14 @@ enum DocLinks {
     return '$url#$hash';
   }
 }
+
+enum DevToolsMode {
+  /// No connection to app and no data from a previous session.
+  disconnected,
+
+  /// Interacting with a connected application.
+  connected,
+
+  /// Showing data saved from a previous session.
+  offlineData,
+}
```

Comment:
hmm. This still seems inaccurate / misleading because these are not disjoint modes. DevTools can be disconnected and showing offline data or connected and showing offline data.

Reply:
Updated comment for offlineData.

DevToolsMode is not about existence of connection, but about what DevTools is doing.